crates/subspace-farmer/src/bin/subspace-farmer/commands/farm.rs

```diff
@@ -234,7 +237,11 @@ pub(crate) async fn bench(
 
     client.stop().await;
 
-    multi_farming.wait().await
+    multi_farming.wait().await?;
+
+    wipe(base_directory)?;
```

Comment:
`TempDir` has `Drop` impl that will delete created directory once dropped, so no need to wipe it explicitly here.

Reply:
Also I don't quite get why you need to crate subdirectory in the first place. You could have used `wipe` on original directly as it only removes known files and directories, I'm a bit confused here.

Reply:
> `TempDir` has `Drop` impl that will delete created directory once dropped, so no need to wipe it explicitly here.

True, but I just thought that it would be better to clean directory explicitly and then just remove directory itself via drop. Will remove `wipe` call then



> Also I don't quite get why you need to crate subdirectory in the first place. You could have used `wipe` on original directly as it only removes known files and directories, I'm a bit confused here.

We need to create another directory as our original directory might have some plot stored in there, so we don't want to reuse and pollute it.